src/System.ValueTuple/src/System.ValueTuple.builds

```diff
@@ -6,6 +6,9 @@
     <Project Include="System.ValueTuple.csproj">
       <TargetGroup>portable-net40+sl4+win8+wp8</TargetGroup>
     </Project>
+    <Project Include="System.ValueTuple.csproj">
```

Comment:
You will get an error on multiple fronts one for compiling not being able to select the right assets and packaging not knowing the correct data.